specification/network/resource-manager/Microsoft.Network/2017-09-01/loadBalancer.json

```diff
@@ -1067,10 +1067,11 @@
         },
         "protocol": {
           "type": "string",
-          "description": "The transport protocol for the external endpoint. Possible values are 'Udp' or 'Tcp'",
+          "description": "The transport protocol for the external endpoint. Possible values are 'Udp' or 'Tcp' or 'All.'",
           "enum": [
             "Udp",
-            "Tcp"
+            "Tcp",
+            "All"
```

Comment:
There are two more definitions of `TransportProtocol` enums. The new `All` value makes this definition different from others. I would suggest us one common definition of `TransportProtocol`. Something like `#/definitions/TransportProtocol`.